### PR TITLE
Fix dependency for RCurl.

### PR DESCRIPTION
**Pull request by davebx**
And also update the setup_r_environment to download mirrored tarballs.

### CODE REVIEW

File: packages/package_cummerbund_2_8_2/tool_dependencies.xml [hunk 1]
```diff
@@ -10,70 +13,73 @@
...
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/biomaRt_2.22.0.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/GenomicAlignments_1.2.1.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/rtracklayer_1.26.2.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/BSgenome_1.34.1.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/GenomicFeatures_1.18.3.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/VariantAnnotation_1.12.7.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/biovizBase_1.14.1.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/Gviz_1.10.3.tar.gz</package>
-                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/cummeRbund_2.8.2.tar.gz</package>
```
Comment (erasche) [inline packages/package_cummerbund_2_8_2/tool_dependencies.xml]: All of the bioconductor packages can/should be replaced with bioarchive copies. 
See http://bioarchive.s3-website-us-east-1.amazonaws.com/

https://github.com/galaxyproject/tools-iuc/issues/128

Comment (erasche) [inline packages/package_cummerbund_2_8_2/tool_dependencies.xml]: (I like depot packages and all, but there's a huge difference between the very private galaxy depot, and the community contributed bioaRchive which is specifically for archiving bioconductor packages, and every single version of them at that.)

**Comment by davebx** [comment]
@erasche I got all the bioarchive ones that actually existed, the rest will have to wait.

**Comment by bgruening** [comment]
@erasche can we merge this?

**Comment by erasche** [comment]
@bgruening it was merged 5 days ago?

**Pull request merged**